core/i18n/resources/en.js

```diff
@@ -240,11 +240,11 @@ module.exports = {
 
       // analysis
       [ActivityLog.type.processingChainCreate]: 'added processing chain',
-      [ActivityLog.type.processingChainPropUpdate]: 'updated {{prop}} processing chain {{label}}',
+      [ActivityLog.type.processingChainPropUpdate]: 'updated {{key}} of processing chain {{label}}',
       [ActivityLog.type.processingChainDelete]: 'deleted processing chain {{label}}',
       [ActivityLog.type.processingStepCreate]: 'added processing step {{index}} to processing chain {{processingChainLabel}}',
-      [ActivityLog.type.processingStepPropsUpdate]: 'updated processing step {{index}} {{prop}} to {{value}}',
-      [ActivityLog.type.processingStepDelete]: 'deleted processing step {{index}}',
+      [ActivityLog.type.processingStepPropsUpdate]: 'updated processing step {{index}} of processing chain {{processingChainLabel}}',
```

Comment:
I would say that this message is rather: ```'updated {{key}} {{value}} of processing chain {{processingChainLabel}} step {{index}}'```
where {{key}} is entity || category and {{value}} is entity label or category name.
what do you think?

Reply:
Good idea, but we pass the entire props object to the update method, we need to guess what key is being updated or modify that function and pass key and value parameters.

Reply:
yes, entityUuid is valid when categoryUuid is null and vice versa. you just need to check which one is populated

Reply:
ok

Reply:
I implemented the merge props in the repository; in the client I set to null the one that's not used. this is still work in progress, we'll see how it will evolve